web/styles/dark_theme.css

```diff
@@ -407,11 +407,6 @@
         box-shadow: 0 0 5px hsl(0deg 0% 100% / 40%);
     }
 
-    .dropdown-widget-button {
-        border-color: hsl(0deg 0% 0% / 60%);
```

Comment:
What about the colors here? I don't see those in the CSS variables in the final commit. If we're using new colors, I'd just include these changes in part of that commit, and note the color change (and source).

Reply:
This commit was accidentally left in this PR when I was creating different PRs for refactoring colors in `dark_theme.css`.